chat-widget/src/components/livechatwidget/common/endChat.ts

```diff
@@ -19,6 +19,9 @@ import { TelemetryManager } from "../../../common/telemetry/TelemetryManager";
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 const prepareEndChat = async (props: ILiveChatWidgetProps, chatSDK: any, state: ILiveChatWidgetContext, dispatch: Dispatch<ILiveChatWidgetAction>, setAdapter: any, setWebChatStyles: any, adapter: any) => {
     try {
+        //Get new auth token again if chat continued for longer time, otherwise chatsdk throws 401 error for auth chats
+        await handleAuthenticationIfEnabled(props, chatSDK);
```

Comment:
If token is not expired, will this hurt performance?

Reply:
Yes and No, 
Yes - because it's making an additional call to get token. 
No - because chat is already ending at this stage.

Should we allow this PR to fix the breaking functionality and have another tracking item for perf improvement?

Reply:
I was reviewing the code again, if performance is a problem, then instead of introducing a new `handleAuthenticationIfEnabled`, maybe we should move the existing one from `endChat` to this method in `prepareEndChat`?